app/mm-redux/constants/index.ts

```diff
@@ -2,7 +2,7 @@
 // See LICENSE.txt for license information.
 import General from './general';
 import RequestStatus from './request_status';
-import WebsocketEvents from './websocket';
+import {WebsocketEvents} from '@constants';
```

Comment:
Why do we need this.. we should change the reference where `WebsocketEvents` is being used instead.

Reply:
You're right! Handled via 6766e1f64055a9fa705c192ce1b74295bd04a269 . All references were already moved.